index.js

```diff
@@ -49,7 +49,7 @@ var WinstonCloudWatch = function(options) {
       config = assign(config, options.awsOptions);
     }
 
-    this.cloudwatchlogs = new CloudWatchLogs(config);
+    this.cloudwatchlogs = new CloudWatchLogs({credentials:config});
```

Comment:
How are you instantiating the `WinstonCloudWatch` transport? When I do so via:
```javascript
new WinstonCloudWatch({
  ...,
  awsAccessKeyId,
  awsSecretKey,
  awsRegion,
})
```
This fix doesn't work. Specifically because in [`CloudWatchLogsClientConfig`](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/clients/client-cloudwatch-logs/interfaces/cloudwatchlogsclientconfig.html#credentials), `region` is specified at the same level of `credentials`. 

Additionally, I believe this proposed fix would break the transport for those who:
1. only specify region and allow AWS SDK to pull creds from the IAM role
2. specify the `CloudWatchLogsClientConfig` directly though `awsOptions`

Thus, I would propose something like changing line 40 to:
```javascript
config = {
  credentials: {
    accessKeyId: awsAccessKeyId,
    secretAccessKey: awsSecretKey,
  },
  region: awsRegion
};
```

And this line back to its original
```javascript
this.cloudwatchlogs = new CloudWatchLogs(config);
```

Reply:
@lazywithclass yes @nikhilrajaram is right according to the docs, I didn't notice that issue because all of my test cases were in the same region.